src/Adapter/MSTest.CoreAdapter/Discovery/TypeEnumerator.cs

```diff
@@ -127,6 +127,7 @@ internal Collection<UnitTestElement> GetTests(ICollection<string> warnings)
         /// <returns> Returns a UnitTestElement.</returns>
         internal UnitTestElement GetTestFromMethod(MethodInfo method, bool isDeclaredInTestTypeAssembly, ICollection<string> warnings)
         {
+            // null if the current instance represents a generic type parameter.
             Debug.Assert(this.type.AssemblyQualifiedName != null, "AssemblyQualifiedName for method is null.");
```

Comment:
What do we need to change the below Debug.Assert as well to reflect that?

Reply:
That was the only documented instance I could find for that. Added the comment for future clarification.